### ISSUE TITLE
Ludovico Technique should automatically reset and cap the minimum shot speed to 1.0 if the shot speed stat is below 1.

### ISSUE THREAD

**Issue by crossstoney** [UnderscoreKilburn/repentance-issues#1248]
One of the major issues of Ludovico Technique is that it relies on shot speed for quicker movement, and shot speed isn't an easy stat to boost normally. I think that therefore, Ludovico Technique should reset lower shot speed to 1 and prevent it from going any lower.

**Comment by Rchardon** [comment]
Just don't take Ludovico if you have low shot speed ??

**Comment by crossstoney** [comment]
Yeah but that’s assuming that one is always aware that they’re getting Ludo, which sometimes isn’t the case (like Curse of the Blind). And honestly though, why not have a shot speed cap with Ludo? It already significantly alters gameplay style and has its negatives anyways even with a shot speed cap. This is merely a suggestion though.
